zenoh-jni/src/key_expr.rs

```diff
@@ -29,22 +30,17 @@ pub(crate) unsafe extern "C" fn Java_io_zenoh_jni_JNIKeyExpr_00024Companion_tryF
     mut env: JNIEnv,
     _class: JClass,
     key_expr: JString,
-) -> *const KeyExpr<'static> {
-    let key_expr_str = match decode_string(&mut env, key_expr) {
-        Ok(key_expr) => key_expr,
-        Err(err) => {
-            _ = err.throw_on_jvm(&mut env);
-            return null();
-        }
-    };
-    let key_expr = match KeyExpr::try_from(key_expr_str) {
-        Ok(key_expr) => key_expr,
-        Err(err) => {
-            _ = Error::KeyExpr(err.to_string()).throw_on_jvm(&mut env);
-            return null();
-        }
-    };
-    Arc::into_raw(Arc::new(key_expr))
+) -> jstring {
```

Comment:
nit: `JString` vs jstring, are similarly named. 
looks like `jstring` is a `*mut _jobject`
while `JString` is a `pub struct JString<'local>(JObject<'local>);`
and a `_jobject` is an empty enum 
`pub enum _jobject {}` 

Does this function the return an empty enum ?
I want to better understand